server/scsynth/SC_Lib.cpp

```diff
@@ -54,6 +54,60 @@ void SendDoneWithIntValue(ReplyAddress *inReply, const char *inCommandName, int
 	SendReply(inReply, packet.data(), packet.size());
 }
 
+static void appendVarArgs(small_scpacket& packet, const char *oscFormat, va_list args)
+{
+	// support all the types the server currently does
+	while (*oscFormat != '\0') {
+		if (*oscFormat == 'i') {
```

Comment:
Could these be a switch statement instead?

Reply:
doesn't seem to be an optimization for commonly occurring matches here, so i think a switch is theoretically better, and would read more cleanly too. i'd also prefer a `for` with an empty first statement over a `while` with added increment at the end, but that's just me. what do you think @muellmusik ?